transformer_engine/pytorch/attention.py

```diff
@@ -2212,10 +2373,17 @@ def forward(
 
     @staticmethod
     def backward(ctx, dout):
+        if ctx.cp_group_a2a is not None:
+            cp_size_a2a = get_distributed_world_size(ctx.cp_group_a2a)
+            rank_a2a = get_distributed_rank(ctx.cp_group_a2a)
+        else:
+            cp_size_a2a = 1
+            rank_a2a = 0
```

Comment:
Could we save `cp_size_a2a` and `rank_a2a` from forward, or is it necessary to call `get_distributed_world_size` and `get_distributed_rank` every time?

Reply:
fixed